libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html

```diff
@@ -1,10 +1,18 @@
-<span class="sky-toggle-switch">
+<span
+  class="sky-toggle-switch"
+  [ngClass]="{
+    'sky-toggle-switch-with-label': hasLabelComponent
+  }"
+>
   <button
     class="sky-toggle-switch-button"
     role="switch"
     type="button"
+    skyId
+    #toggle="skyId"
```

Comment:
Template reference variables should be placed at the end of the tag.

Reply:
Moved.